Problems/prob061/specification.md

```diff
@@ -0,0 +1,25 @@
+---
+Title:    Resource-Constrained Project Scheduling Problem (RCPSP)
+Proposer: Peter Nightingale and Emir Demirović
```

Comment:
Should be written as 

Proposer:
  - Peter Nightingale
  - Emir Demirović

So that problem appears under both of the names.